lib/widgets/countdown_section.dart

```diff
@@ -0,0 +1,175 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:tag_highlighting/tag_highlighting.dart';
+
+import 'package:hackathon_flutterturkiye_org/core/styles/theme_helper.dart';
+import 'package:hackathon_flutterturkiye_org/core/utils/responsive_helper.dart';
+import 'package:hackathon_flutterturkiye_org/widgets/responsive_builder.dart';
+
+const double _fontSizeSmall = 24.0;
+const double _fontSizeMedium = 48.0;
+const double _fontSizeLarge = 72.0;
+
+const double _paddingSmall = 24.0;
+const double _paddingMedium = 48.0;
+const double _paddingLarge = 72.0;
+
+final DateTime _eventStartingDate = DateTime(2021, 3, 6, 10, 0);
+
+class CountdownSection extends StatelessWidget {
+  const CountdownSection();
+
+  @override
+  Widget build(BuildContext context) {
+    final screenSize = MediaQuery.of(context).size;
+    return Container(
+      decoration: BoxDecoration(color: Colors.black),
+      height: screenSize.height,
+      padding: _padding(context),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          const CountdownTitle(),
+          SizedBox(height: screenSize.height * 0.1),
+          const CountdownWidget(),
+        ],
+      ),
+    );
+  }
+
+  EdgeInsets _padding(BuildContext context) {
+    if (ResponsiveHelper.isLargeScreen(context)) {
+      return EdgeInsets.symmetric(horizontal: _paddingLarge);
+    } else if (ResponsiveHelper.isMediumScreen(context)) {
+      return EdgeInsets.symmetric(horizontal: _paddingMedium);
+    } else {
+      return EdgeInsets.symmetric(horizontal: _paddingSmall);
+    }
+  }
+}
+
+class CountdownTitle extends StatelessWidget {
+  const CountdownTitle();
+
+  final String text =
+      "Türkiye'nin en büyük\n<bold>Flutter Festivali</bold> başlıyor";
+
+  @override
+  Widget build(BuildContext context) {
+    return _CountdownText(text: text);
+  }
+}
+
+class CountdownWidget extends StatefulWidget {
+  const CountdownWidget();
+
+  @override
+  _CountdownWidgetState createState() => _CountdownWidgetState();
+}
+
+class _CountdownWidgetState extends State<CountdownWidget> {
+  Timer _timer;
+
+  @override
+  void initState() {
+    super.initState();
+    _setTimer();
+  }
+
+  @override
+  void dispose() {
+    _timer.cancel();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final currentDate = DateTime.now();
+    final remainingTime = _eventStartingDate.difference(currentDate);
+    final text = '<bold>${remainingTime.inDays}</bold> gün '
```

Comment:
Why don't you use RichText here?

Reply:
Because, the tag highlighter package is doing all these works for me.

Reply:
But that is wrong :) you just pass in markdown and rely on library for something that you can do out of box.

Reply:
Done.